.env.example

```diff
@@ -5,4 +5,5 @@ NODE_ENV="development"
 
 GITHUB_ID=""
 GITHUB_SECRET=""
-NEXTAUTH_SECRET=""
+NEXTAUTH_SECRET="123456"
```

Comment:
Question for discussion (i do not know the answer) - can this value be anything or is it a better practice to generate it another way?  I know it's completely visible so I'm assuming it does not matter, but in the docs it's suggested to generate a "good" value like this: 

You can quickly create a good value on the command line via this openssl command.
```$ openssl rand -base64 32```

Reply:
ah, yes - you're correct. I missed that. Perfect.

Reply:
yep in prod we use that command to generate a more secure secret